frontend/src/components/activity/ActivityContent.vue

```diff
@@ -29,18 +29,36 @@
 
         <div
           v-if="editInstanceName"
-          style="flex: 1;"
-          @click.stop
-          @keyup.prevent>
-          <api-text-field
-            dense
-            autofocus
-            :uri="activityContent._meta.self"
-            fieldname="instanceName" />
+          style="display: flex; flex: 1; align-items: center"
+          @click.stop>
+          <div
+            style="flex: 1"
```

Comment:
```suggestion
            class="flex-grow-1"
```
Ausserdem kannst du den `style="flex: 0 0 30px"` von den Buttons wegnehmen.